src/pages/MyPage/MyPage.module.css

```
@@ -18,7 +18,7 @@
   flex-direction: column;
   min-height: 100vh;
   background: var(--bg);
-  font-family: 'Nunito', sans-serif;
+  font-family: "Nunito", sans-serif;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Nunito" (font-family-name-quotes)

(font-family-name-quotes)